data/en/1_0215.json

```diff
@@ -23,269 +23,602 @@
+                                    "retail sales from outlets in Great Britain to customers abroad"
+                                ]
                             },
                             {
-                                "answers": [{
-                                    "guidance": "<div> <h4>Include</h4> <ul> <li>VAT</li> <li>internet sales</li> <li>retail sale from outlets in Great Britain to customers abroad</li> </ul> </div> <div> <h4>Exclude</h4> <ul> <li>revenue from mobile phone network commission and top up </li> <li>sales from catering facilities used by customers</li>  <li>lottery sales and commission from lottery sales</li> <li>sales of car accessories and motor vehicles</li> <li>NHS receipts</li> </ul> </div>",
-                                    "id": "total-retail-turnover",
-                                    "label": "What was the value of the business\u2019s total retail turnover?",
-                                    "mandatory": true,
-                                    "options": [],
-                                    "q_code": "20",
-                                    "type": "Currency",
-                                    "validation": {
-                                        "messages": {
-                                            "INTEGER_TOO_LARGE": "The maximum value allowed is 9999999999. Please correct your answer.",
-                                            "MANDATORY": "Please provide a value, even if your value is 0.",
-                                            "NEGATIVE_INTEGER": "The value cannot be negative. Please correct your answer.",
-                                            "NOT_INTEGER": "Please only enter whole numbers into the field."
-                                        }
-                                    }
-                                }],
-                                "description": "",
-                                "id": "total-retail-turnover-question",
-                                "title": "",
-                                "type": "General"
+                                "title": "Exclude",
+                                "list": [
+                                    "revenue from mobile phone network commission and top-up",
+                                    "sales from catering facilities used by customers",
+                                    "lottery sales and commission from lottery sales",
+                                    "sales of car accessories and motor vehicles",
+                                    "NHS receipts"
+                                ]
+                            }
+                        ],
+                        "answers": [{
+                            "guidance": "",
+                            "id": "total-retail-turnover",
+                            "alias": "total_turnover",
+                            "description": "<p>Round to the nearest (\u00a3) pound. Do not  include pence<p>",
```

Comment:
do not put these answer descriptions in `<p>` tags otherwise we're getting strange spacing issues; applies for all below as well